.travis.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       script: ./gradlew javadocs
 
     - stage: snapshot
-      if: branch = master AND NOT (type = pull_request)
+      if: (branch = master OR branch =~ /^r[0-9]+\.[0-9]+/) AND NOT (type = pull_request)
```

Comment:
This won't work for `r0.10` or `r0.11`. You might want to take care of that too.

Reply:
Actually, this'd work, as there are `+` symbols in the pattern.